examples/desec.rs

```diff
@@ -0,0 +1,35 @@
+use dns_update::{DnsRecord, DnsRecordType, DnsUpdater};
+use std::env;
+use std::time::Duration;
+
+#[tokio::main]
+pub async fn main() -> Result<(), std::env::VarError> {
+    let token = env::var("DESEC_TOKEN").expect("Envvar DESEC_TOKEN should be set with valid token");
+    let domain =
+        env::var("DESEC_DOMAIN").expect("Envvar DESEC_DOMAIN should be set with DNS domain");
+
+    let client = DnsUpdater::new_desec(token, Some(Duration::from_secs(120))).unwrap();
+
+    // Create a new TXT record
+
+    let client_result = client
+        .create(
+            format!("_domainkey.{}", domain),
+            DnsRecord::TXT {
+                content: "\"v=DKIM1; k=rsa; h=sha256; p=test\"".to_string(),
```

Comment:
When https://github.com/stalwartlabs/dns-update/pull/21 is committed are the extra `\"` no more requierd